types/registry.go

```diff
@@ -0,0 +1,66 @@
+package types
+
+import "time"
+
+// CompanyAddress represents a company address
+type CompanyAddress struct {
+	AdressLine1 string `json:"address-line-1"`
+	AdressLine2 string `json:"address-line-2"`
+	AdressLine3 string `json:"address-line-3"`
+	AdressLine4 string `json:"address-line-4"`
+	PostCode    string `json:"post-code"`
+	Country     string `json:"country"`
+	CareOf      string `json:"care-of"`
+	Type        string `json:"type"`
+}
+
+// CompanyAnnualReturn represents a company's annual return details
+type CompanyAnnualReturn struct {
+	FilingMonth int       `json:"filing-month"`
+	LastFiled   time.Time `json:"last-filed"`
+}
+
+// CompanyDirector represents a company director
+type CompanyDirector struct {
+	Name        string    `json:"name"`
+	AppointedAt time.Time `json:"appointed-at"`
+	Status      string    `json:"status"`
+}
+
+// CompanyShareAllocation represents a company share allocation
+type CompanyShareAllocation struct {
+	Type       string `json:"type"`
+	Allocation int    `json:"allocation"`
+	Name       string `json:"shareholder-name"`
```

Comment:
Should we shorten the `shareholder-name` in this struct to just `shareholder` (or just `name`)?

Reply:
Changed to `sharholder`